packages/core/components/DirectoryTree/useDirectoryHierarchy.tsx

```diff
@@ -209,6 +187,9 @@ const useDirectoryHierarchy = (
                             take(pathToChildNode, depth + 1)
                         ).map((pair) => {
                             const [name, value] = pair as [string, string];
+                            if (value === NO_VALUE_NODE) {
+                                return new ExcludeFilter(name);
```

Comment:
Could `FileFilter` auto construct an `ExcludeFilter` when the `value` is `NO_VALUE_NODE `? I think it makes things cleaner when this kind of logic lives closer to the domain it is helping.

Reply:
Changed in [547aed0](https://github.com/AllenInstitute/biofile-finder/pull/498/commits/547aed022256a7672ae3c741610098864a8bd1ec)